src/main/resources/db/changelog/20220722000000-add_delete_cascade_to_consumer_fks.xml

```diff
@@ -6,56 +6,93 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
-    <changeSet id="20220613000000-1" author="jalbrech">
+    <changeSet id="20220722000000-1" author="jalbrech">
+        <preConditions onFail="MARK_RAN">
+            <foreignKeyConstraintExists foreignKeyName="cp_content_tags_consumer_fk"/>
+        </preConditions>
 
         <dropForeignKeyConstraint  baseTableName="cp_consumer_content_tags" constraintName="cp_content_tags_consumer_fk"/>
-        <dropForeignKeyConstraint  baseTableName="cp_consumer_facts" constraintName="fk278ce810390317d"/>
-        <dropForeignKeyConstraint  baseTableName="cp_consumer_capability" constraintName="fk_cnsmr_capability_cnsmr"/>
-        <dropForeignKeyConstraint  baseTableName="cp_consumer_guests" constraintName="fk_consumer_guests"/>
-        <dropForeignKeyConstraint  baseTableName="cp_installed_products" constraintName="fk_installed_product"/>
-        <dropForeignKeyConstraint  baseTableName="cp_consumer_guests_attributes" constraintName="fk_guest_attribute"/>
 
         <addForeignKeyConstraint baseTableName="cp_consumer_content_tags"
             baseColumnNames="consumer_id"
             referencedTableName="cp_consumer"
             referencedColumnNames="id"
             constraintName="cp_content_tags_consumer_fk"
             onDelete="CASCADE" />
+    </changeSet>
+
+    <changeSet id="20220722000000-2" author="jalbrech">
+        <preConditions onFail="MARK_RAN">
+            <foreignKeyConstraintExists foreignKeyName="fk278ce810390317d"/>
+        </preConditions>
+
+        <dropForeignKeyConstraint  baseTableName="cp_consumer_facts" constraintName="fk278ce810390317d"/>
 
         <addForeignKeyConstraint baseTableName="cp_consumer_facts"
             baseColumnNames="cp_consumer_id"
             referencedTableName="cp_consumer"
             referencedColumnNames="id"
-            constraintName="fk278ce810390317d"
+            constraintName="cp_consumer_facts_consumer_fk"
             onDelete="CASCADE" />
-        
+    </changeSet>
+
+    <changeSet id="20220722000000-3" author="jalbrech">
+        <preConditions onFail="MARK_RAN">
+            <foreignKeyConstraintExists foreignKeyName="fk_cnsmr_capability_cnsmr"/>
+        </preConditions>
+
+        <dropForeignKeyConstraint  baseTableName="cp_consumer_capability" constraintName="fk_cnsmr_capability_cnsmr"/>
```

Comment:
All the drops have extra whitespace between the command and the table name. Is this intentional?

Reply:
Good catch, that wasn't intentional. Removed the extra whitespace